pkg/api/types.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	// variables.
 	EnvironmentFile string
 
+	// BuildNamespace provides the namespace under which the labels will be generated.
+	BuildNamespaceLabel string
```

Comment:
Hey, the value of that field is the namespace of the label, not the label of the namespace, therefore should be `NamespaceLabel`.